core/src/main/kotlin/com/karumi/kotlinsnapshot/core/Camera.kt

```diff
@@ -81,15 +97,23 @@ internal class Camera<in A>(
         }
     }
 
-    private fun extractTestCaseName(): String {
+    private fun extractTestCaseName(): TestCaseName {
         val testCaseTrace = extractor.getTestStackElement()
         if (testCaseTrace != null) {
-            return "${testCaseTrace.className}_${testCaseTrace.methodName}"
+            return TestCaseName(testCaseTrace.className, testCaseTrace.methodName)
         } else {
             throw TestNameNotFoundException("Kotlin Snapshot library couldn't find the name " +
                 "of the test. Review if the test case file or the spec file contains the word " +
                 "test or spec or specify a snapshot name manually, this is a requirement needed " +
                 "to use Kotlin Snapshot")
         }
     }
+
+    private data class TestCaseName(val testClassName: String?, val methodName: String) {
```

Comment:
Minor: `TestCaseName` already implies that we are dealing with Tests so I'd rather use `className` instead of `testClassName`.

Reply:
you are right, or even if I would like the idea of use testWhatever.. why is it `testClassName` and no `testMethodName`? 🤔 I'm going to change it, thanks!